src/content/docs/apm/agents/ruby-agent/features/ruby-codestream-integration.mdx

```diff
@@ -14,7 +14,7 @@ Display context-sensitive APM data directly in your IDE by integrating [New Reli
 First, [install](/docs/codestream/start-here/install-codestream) the New Relic CodeStream extension into your supported IDE of choice and log in.
 
 <Callout variant="important">
-  The New Relic CodeStream integration is available in Ruby agent version 8.8.0 and higher and is disabled by default. To enabled the integration, either set `code_level_metrics.enabled = true` in `newrelic.yml` or `NEW_RELIC_APPLICATION_CODE_LEVEL_METRICS_ENABLED=true` as an environment variable.
+  The New Relic CodeStream integration is available in Ruby agent version 8.8.0 and higher and is disabled by default. To enable the integration, either set `code_level_metrics.enabled = true` in `newrelic.yml` or `NEW_RELIC_APPLICATION_CODE_LEVEL_METRICS_ENABLED=true` as an environment variable.
```

Comment:
@fallwith, thanks for fixing that typo here! Also, should the environment variable be `NEW_RELIC_CODE_LEVEL_METRICS_ENABLED` in this context?

Reply:
Thank you, @rhetoric101 - addressed with 3d0be60

How embarrassing! It would be one thing if the typo existed in a file elsewhere in the repo but not already included in this PR, but it was right there in the PR in a small section that contained an update! How silly of me not to notice. Thank you for catching it!